plugins/ckan_operators/datastore_operator.py

```diff
@@ -119,6 +119,58 @@ def execute(self, context):
         return result
 
 
+
```

Comment:
FYI there is already a DeleteDatastoreResourceRecordsOperator, but it requires a backup_task_id as an input. I didnt want to repurpose it since its already widely used, but I definitely could combine them into a single operator down the road.

If you think it's worthwhile, let me know and Ill add a card to Trello